src/navigation-plugin.tsx

```diff
@@ -198,6 +199,28 @@ export class NavigationPlugin extends KalturaPlayer.core.BasePlugin {
   private _makeQuizTitle = (state: number, index: number, type: number) => {
     return <QuizTitle questionState={state} questionIndex={index} questionType={type} />;
   };
+  private _makeQuizTitleAriaLabel = (state: number, index: number, type: number, question: string): string => {
+    const currentIndex = index + 1;
+
+    let label = '';
+    if (type === 3) {
```

Comment:
you can use a switch instead of an if if the condition is simply checking a value